README.md

```diff
@@ -14,99 +14,30 @@ Add your own `fetch` library.
 * for browsers use [whatwg-fetch](https://www.npmjs.com/package/whatwg-fetch) as polyfill for Internet Explorer
 * in AWS Lambda functions use [node-fetch](https://www.npmjs.com/package/node-fetch)
 
-> In browser enviroments
-> copy ` dist/bima-shark-sdk.js ` into your app.
-
+> In browser enviroments:
+> - run `npm run build`
+> - copy ` dist/bima-shark-sdk.js ` into your app.
 
 ### Usage
 
 #### Configuration (optional)
 
 ```
+import Shark from 'bima-shark-sdk'
+
+
```

Comment:
one line too many, standard.js Is not screaming?

Reply:
Ok, forgot about `standard.js`. Thanks for the reminder.